### PR TITLE
chore(deps): bump aws-cdk from 1.70.0 to 1.72.0

### PR DESCRIPTION
**Pull request by kozlove-aws**
### Notes
- Updates the CDK version from 1.70.0 to 1.72.0
- Updates the dependencies of tools shared with CDK

## Testing 
- Ran it through a CodePipeline and ensured it succeeded.
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by ddneilson — CHANGES_REQUESTED** [review]
From the build script:
```
warning " > @aws-cdk/aws-events-targets@1.72.0" has unmet peer dependency "@aws-cdk/aws-kinesisfirehose@1.72.0".
```

It looks like another dependency got added between CDK packages.


Also:

```
 aws-rfdk:   ● Console
aws-rfdk:     console.log
aws-rfdk:       Failed to validate the version string: version
aws-rfdk:       at VersionProviderResource.implementsIVersionProviderResourceProperties (lib/core/lambdas/nodejs/version-provider/handler.ts:129:17)
```

It's possible that this later one is not related to the CDK change, but it would be worth confirming that.

**Comment by kozlove-aws** [comment]
> Failed to validate the version string: version
Is expected message and not connected to CDK change. Probably we should clean it but not here.

**Comment by horsmand** [comment]
> Also:
> 
> ```
>  aws-rfdk:   ● Console
> aws-rfdk:     console.log
> aws-rfdk:       Failed to validate the version string: version
> aws-rfdk:       at VersionProviderResource.implementsIVersionProviderResourceProperties (lib/core/lambdas/nodejs/version-provider/handler.ts:129:17)
> ```
> 
> It's possible that this later one is not related to the CDK change, but it would be worth confirming that.

This logging is from my recent commit for the VersionQuery and happens in a unit test for the version provider Lambda that is testing a bad version string in the `implementsIVersionProviderResourceProperties` method. The test expects the version string to fail the validation, so the logging is expected, but it should probably be suppressed in the unit test. I'll look into that separately, this shouldn't hold up this PR.

File: package.json [hunk 1]
```diff
@@ -26,74 +26,75 @@
...
+    "@types/jest": "^26.0.15",
+    "@types/node": "^14.14.6",
     "@types/nodeunit": "^0.0.31",
-    "aws-cdk": "1.70.0",
-    "aws-sdk": "^2.741.0",
-    "constructs": "^3.0.4",
-    "conventional-changelog-cli": "^2.1.0",
+    "aws-cdk": "1.72.0",
+    "aws-sdk": "^2.787.0",
+    "constructs": "^3.2.15",
```
Comment (horsmand) [inline package.json]: CDK goes to a lot of effort to make sure they are pinning the version of `constructs` in all of their packages. I think we should try to always match their version and keep our linting rule the same as theirs. It looks like we missed this rule update in `pkglint` from 11 days ago: https://github.com/aws/aws-cdk/blob/master/tools/pkglint/lib/rules.ts#L1326

Our rule that needs updating is here: https://github.com/aws/aws-rfdk/blob/mainline/tools/pkglint/lib/rules.ts#L928

Comment (kozlove-aws) [reply package.json]: Ok, I have changed `constructs` to version that used in CDK 3.2.0 instead of last version 3.2.15

**Review by horsmand — APPROVED** [review]

**Review by ddneilson — APPROVED** [review]

**Pull request merged**